src/pages/ReportDescriptionPage.js

```diff
@@ -80,39 +80,39 @@ function ReportWelcomeMessagePage(props) {
         <ScreenWrapper
             shouldEnableMaxHeight
             includeSafeAreaPaddingBottom={false}
-            testID={ReportWelcomeMessagePage.displayName}
+            testID={ReportDescriptionPage.displayName}
         >
-            <FullPageNotFoundView shouldShow={ReportUtils.shouldDisableWelcomeMessage(props.report, props.policy)}>
+            <FullPageNotFoundView shouldShow={ReportUtils.shouldDisableReportDescription(props.report, props.policy)}>
                 <HeaderWithBackButton
-                    title={props.translate('welcomeMessagePage.welcomeMessage')}
+                    title={props.translate('reportDescriptionPage.roomDescription')}
                     onBackButtonPress={() => Navigation.goBack(ROUTES.REPORT_SETTINGS.getRoute(props.report.reportID))}
                 />
                 <FormProvider
                     style={[styles.flexGrow1, styles.ph5]}
-                    formID={ONYXKEYS.FORMS.WELCOME_MESSAGE_FORM}
+                    formID={ONYXKEYS.FORMS.REPORT_DESCRIPTION}
                     onSubmit={submitForm}
                     submitButtonText={props.translate('common.save')}
                     enabledWhenOffline
                 >
-                    <Text style={[styles.mb5]}>{props.translate('welcomeMessagePage.explainerText')}</Text>
+                    <Text style={[styles.mb5]}>{props.translate('reportDescriptionPage.explainerText')}</Text>
                     <View style={[styles.mb6]}>
                         <InputWrapper
                             InputComponent={TextInput}
-                            inputID="welcomeMessage"
-                            label={props.translate('welcomeMessagePage.welcomeMessage')}
-                            accessibilityLabel={props.translate('welcomeMessagePage.welcomeMessage')}
+                            inputID="reportDescription"
+                            label={props.translate('reportDescriptionPage.roomDescription')}
+                            accessibilityLabel={props.translate('reportDescriptionPage.roomDescription')}
                             role={CONST.ROLE.PRESENTATION}
                             autoGrowHeight
                             maxLength={CONST.MAX_COMMENT_LENGTH}
                             ref={(el) => {
                                 if (!el) {
                                     return;
                                 }
-                                welcomeMessageInputRef.current = el;
-                                updateMultilineInputRange(welcomeMessageInputRef.current);
+                                reportDescriptionInputRef.current = el;
```

Comment:
Bug: navigate to an existing workspace room that doesn't have description.
It shows not found page.

https://github.com/Expensify/App/assets/29673073/2e7d3f30-225a-400e-aa77-ca251407d1b8

Reply:
@rushatgabhane Currently we are showing the `notFound` page for those reports which are being excluded in this function:
<img width="1134" alt="Screenshot 2024-01-14 at 10 10 45 PM" src="https://github.com/Expensify/App/assets/77237602/c11fb308-1662-4900-a103-eabf1655a891">

But I agree we need to change this. I'll make some modifications now

Reply:
Since this is a policy expense chat, there shouldn't be any room description. We are working here: https://github.com/Expensify/App/issues/32955#issue-2038816407 to show only the policy description for the policy expense chat.

Reply:
Here: https://github.com/Expensify/App/pull/34150/commits/e28f6cddaf021ce341d75b4a631b1a5a5ffaa3a4
I made the change that description is not visible for a report if:
- The User cannot edit description AND description is empty
But visible if:
- The user can edit OR description is not empty.
This way I don't think there will be a not found page in the flow.